bin/spiced/Cargo.toml

```diff
@@ -21,11 +21,12 @@ tracing-subscriber.workspace = true
 metrics-exporter-prometheus = "0.13.0"
 
 [features]
-default = ["duckdb", "postgres", "sqlite", "mysql", "flightsql", "databricks", "dremio"]
+default = ["duckdb", "postgres", "sqlite", "mysql", "flightsql", "databricks", "dremio", "clickhouse"]
 duckdb = ["runtime/duckdb", "spicepod/duckdb", "app/duckdb"]
 postgres = ["runtime/postgres", "spicepod/postgres", "app/postgres"]
 sqlite = ["runtime/sqlite", "spicepod/sqlite", "app/sqlite"]
 mysql = ["runtime/mysql", "spicepod/mysql", "app/mysql"]
+clickhouse = ["runtime/clickhouse", "spicepod/clickhouse", "app/clickhouse"]
```

Comment:
You will need to add to the conditional feature build too.

Reply:
What conditional feature build?
Didn't found any relevant fields in the config file

Reply:
Here: https://github.com/spiceai/spiceai/blob/trunk/.github/workflows/features.yml

Reply:
Updated